### PR TITLE
fix disabled button hover state

### LINKED ISSUE #852: Fix disabled button hover state
Some disabled buttons are showing state changes on hover when they shouldn't be. Review button disabled css for both default and primary buttons to ensure behavior is the same.

### PR DESCRIPTION
**Pull request by na3d**
Closes #852 

Disabled all mouse events to disabled elements.

### CODE REVIEW

**Comment by eatonphil** [comment]
Does this hide the cursor too? We might not want that if we want click on the disabled button to show unfilled required fields.

**Comment by na3d** [comment]
It makes it so the mouse cursor isn't affected by this button at all.  It shouldn't affect javascript functionality at all.

**Comment by eatonphil** [comment]
That's not what I mean, does it visually hide the cursor that you normally get when you hover over a button?

**Comment by na3d** [comment]
It works the same way disabled buttons normally behave.

**Comment by eatonphil** [comment]
No it does not. Just tried it and it hides the cursor and you cannot fix it with cursor:pointer. This method is unfortunately not going to work.

**Comment by na3d** [comment]
A normal disabled button does the same thing, it doesn't turn into a pointer finger.

**Comment by eatonphil** [comment]
I think this is not the behavior we want in light of something like #727. I think we might want to put off making a decision one way or the other here until we iterate on #727 a bit.

**Comment by na3d** [comment]
I would argue that we should have a different class for that situation, instead of disabling the button.

**Comment by eatonphil** [comment]
Yeah that might be a good point.

File: scss/manager.scss [hunk 1]
```diff
@@ -11,6 +11,10 @@
     transition: background .35s, color .2s, border .1s;
 }
 
+*:disabled {
```
Comment (eatonphil) [inline scss/manager.scss]: You can drop the * here can't you?

Comment (na3d) [inline scss/manager.scss]: No problem.

**Comment by eatonphil** [comment]
@mparke anything to add?

**Comment by mparke** [comment]
yes, so I think the issue to be fixed is just the color changes on hover state, so I'd prefer not to use `pointer events: none`.

There's a `.btn-primary:hover:disabled` style somewhere that's incorrect

**Comment by na3d** [comment]
.btn-primary:hover:disabled or button:hover:disabled?

**Comment by na3d** [comment]
If we are okay with !important being used for now on this, I have a solution.

**Pull request merged**